lib/slack/web/client.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Slack.Web.Client do
```

Comment:
Can you define a behaviour so that other implementations can implement it?

Reply:
ya sure

Reply:
done